packages/vuepress-theme-nimatron/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "@okta/vuepress-theme-nimatron",
+  "version": "0.3.0",
+  "description": "VuePress theme for Odyssey Documentation",
+  "author": "Okta, Inc.",
+  "license": "Apache-2.0",
+  "main": "index.js",
+  "devDependencies": {
+    "node-sass": "4.13.1",
```

Comment:
Odyssey should be a dependency here, right?

Reply:
It should indeed, good catch. Will fix.

Reply:
```suggestion
    "@okta/odyssey": "^0.3.0",
    "node-sass": "4.13.1",
```